### PR TITLE
Release v5.4 changes

### PR DESCRIPTION
**Pull request by Subasri-Viswanathan**
Revamp readme files
Include optional library variables.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,17 +57,36 @@ The above Bold Reports image can be deployed using Docker or Docker Compose. In
   ```sh
   curl -o docker-compose.yml "https://raw.githubusercontent.com/boldreports/bold-reports-docker/master/deploy/single-container-pre-configured/docker-compose.yml"
   ```
-  2. Open the Docker Compose file, fill the BOLD_SERVICES_UNLOCK_KEY value, and save it. You can refer to [this](https://support.boldreports.com/kb/article/13271/how-do-i-get-my-offline-license-key-from-our-bold-reports-account-page) KB document to obtain the offline Bold Reports unlock key.
+  2. Open the Docker Compose file, uncommnet the APP_URL and Replace <App_Url> with your DNS or IP address, by which you want to access the application.
```
Comment (ranganathan-arumugam) [inline README.md:60]: correct the spelling.

File: README.md [hunk 2]
```diff
@@ -57,17 +57,36 @@ The above Bold Reports image can be deployed using Docker or Docker Compose. In
...
 
-     ![docker-compose-variable](docs/images/docker-compose-variable.png)
+      For example,  
+      `http://example.com `  
+      `https://example.com`  
+      `http://<public_ip_address>`
```
Comment (ranganathan-arumugam) [inline README.md:65]: add the port number too.

File: README.md [hunk 3]
```diff
@@ -57,17 +57,36 @@ The above Bold Reports image can be deployed using Docker or Docker Compose. In
...
+  2. Open the Docker Compose file, uncommnet the APP_URL and Replace <App_Url> with your DNS or IP address, by which you want to access the application.
 
-     ![docker-compose-variable](docs/images/docker-compose-variable.png)
+      For example,  
+      `http://example.com `  
+      `https://example.com`  
+      `http://<public_ip_address>`  
+      `http://host.docker.internal`
+
+      Note:
```
Comment (ranganathan-arumugam) [inline README.md:68]: Remove this note section.

File: README.md [hunk 4]
```diff
@@ -57,17 +57,36 @@ The above Bold Reports image can be deployed using Docker or Docker Compose. In
...
+      Use http://host.docker.internal instead of http://localhost. Host machine localhost DNS will not be accessible inside the container. So, docker desktop provides `host.docker.internal` and `gateway.docker.internal` DNS for communication between docker applications and host machine. Please make sure that the host.docker.internal DNS has your IPv4 address mapped in your hosts file on Windows(C:\Windows\System32\drivers\etc\hosts) or Linux (/etc/hosts).
+      Provide the HTTP or HTTPS scheme for APP_BASE_URL value.
+      You can also change the Port number other than 8085.
+
+      ![docker-single-pre-conf](docs/images/single-container-pre-app-url.png)
+
+  3. Fill the BOLD_SERVICES_UNLOCK_KEY value, and save it. You can refer to [this](https://support.boldreports.com/kb/article/13271/how-do-i-get-my-offline-license-key-from-our-bold-reports-account-page) KB document to obtain the offline Bold Reports unlock key. 
+
+      ![docker-compose-variable](docs/images/docker-compose-variable.png)
```
Comment (ranganathan-arumugam) [inline README.md:79]: Take the snap from the vs code with dark mode as like previous one.

File: README.md (hunk 3 above)
Comment (Subasri-Viswanathan) [reply README.md]: Removed the note section

File: README.md (hunk 2 above)
Comment (Subasri-Viswanathan) [reply README.md]: Included the port number

File: README.md (hunk 1 above)
Comment (Subasri-Viswanathan) [reply README.md]: Changed

File: README.md (hunk 4 above)
Comment (Subasri-Viswanathan) [reply README.md:72]: Changed the snap

**Review by ranganathan-arumugam — APPROVED** [review]
Changes are fine. I'm approving this PR.

**Review by Sankaralingam1667 — APPROVED** [review]

**Pull request merged**